### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mikebrow**
just a quick nit fix to lower case containerd

### CODE REVIEW

**Comment by elezar** [comment]
@mikebrow I think you may need to sign-off the commit to satisfy the DCO requirement.

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ command to find them out:
 $ crio config |& grep -B1 -A5 cdi_spec_dirs
 ```
 
-### Containerd configuration
+### containerd configuration
 
 To enable and configure CDI support in the [containerd
```
Comment (fujitatomoya) [inline README.md:80]: how about adding? i ended up looking for this information to try out, which would be useful for other uses?

```suggestion
[cotnainerd v1.7.0](https://github.com/containerd/containerd/releases/tag/v1.7.0) or later required.
To enable and configure CDI support in the [containerd
```

**Review by elezar — APPROVED** [review]
lgtm

**Comment by elezar** [comment]
merged bypassing the DCO check that had not run. Confirmed that the commit was signed-off.

**Pull request merged**